lib/ansible/modules/source_control/git_config.py

```diff
@@ -157,9 +175,10 @@ def main():
             name=dict(type='str'),
             repo=dict(type='path'),
             scope=dict(required=False, type='str', choices=['local', 'global', 'system']),
-            value=dict(required=False)
+            value=dict(required=False),
+            state=dict(required=False, choices=['set', 'unset'])
```

Comment:
Use `present` or `absent` to match almost all other modules

Reply:
The choices have been updated to `present` and `absent`. I also altered the other bits of code affected by this change.